src/commands/mod.rs

```diff
@@ -1435,14 +1435,22 @@ fn log_template(settings: &UserSettings) -> String {
         "styled" => format!("shortest({desired_id_len})"),
         _ => format!("short({desired_id_len})"),
     };
+    let author_template = match settings.log_author_format().as_str() {
+        "none" => None,
+        "full" => Some("author"),
+        "name" => Some("author.name()"),
+        "username" => Some("author.username()"),
+        _ => Some("author.email()"),
+    }
+    .map_or("".to_owned(), |t| t.to_owned() + ",");
```

Comment:
Just fyi, you can use `r#""""#` as an empty value and drop `Option`. That's the point of `separate(" ", ...)` function.

Reply:
Ah, that's pretty nifty. Thanks! (Refactored in #1236)